src/utilities/savedFilters.ts

```diff
@@ -56,12 +56,12 @@ export const oneWeekSavedSearch = new SavedSearch({
 
 export const excludeScheduledSavedSearch = new SavedSearch({
   id: null,
-  name: 'No scheduled',
+  name: 'Hide Scheduled Runs',
```

Comment:
I could be mistaken, but I believe we use sentence case for all the filters
```suggestion
  name: 'Hide scheduled runs',
```